### PR TITLE
cargo.toml: update info fields for packaging

### PR DESCRIPTION
**Pull request by flouthoc**
Update `cargo.toml` with fields necessary for packaging.

### CODE REVIEW

**Comment by flouthoc** [comment]
@mheon @Luap99 @baude @lsm5 PTAL following fields are needed for packaging.

File: Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,12 @@ name = "netavark"
 version = "1.0.4-dev"
 edition = "2018"
 authors = ["github.com/containers"]
+license = "Apache-2.0"
+readme = "README.md"
 description = "A container network stack"
+homepage = "https://github.com/containers/netavark"
+repository = "https://github.com/containers/netavark"
+categories = ["command-line-utilities"]
```
Comment (Luap99) [inline Cargo.toml:11]: netavark is not really a command line utility, is there a fixed list with categories? I would just set `containers` and `networking`.

Comment (flouthoc) [reply Cargo.toml:11]: Its a suggestion in packaging cli manual page not sure if it really matters. But i see other popular CLI tools sharing this category. Let me add `containers` and `networking` as well.

Comment (Luap99) [reply Cargo.toml:11]: Sure but netavark is not used manually by a person so I say it is not a command line utility.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by flouthoc** [comment]
@mheon @lsm5 PTAL

**Pull request merged**